test/functional/framework.go

```diff
@@ -389,6 +399,10 @@ func (f *FluentdFunctionalFramework) Readk8sAuditLogsFrom(outputName string) ([]
 	return f.ReadLogsFrom(outputName, k8sAuditLog)
 }
 
+func (f *FluentdFunctionalFramework) ReadovnAuditLogsFrom(outputName string) ([]string, error) {
```

Comment:
change method name to `ReadOvnAuditLogsFrom` , change `ovn` to `Ovn`

Reply:
done